### PR TITLE
Microservice sdk links

### PR DESCRIPTION
**Pull request by MWindel**
* Renamed files
* Fixed links

### CODE REVIEW

File: content/microservice-sdk/java-bundle/client-library.md [hunk 1]
```diff
@@ -259,7 +260,7 @@ subscriber.disconnect();
 ### Subscribing to Notifications 2.0
 
 The Notifications 2.0 API can be accessed in a very similar manner as described above in [Accessing the inventory](#accessing-the-inventory).
-See [Notifications 2.0](/reference/notifications) in the *Reference guide* for more details about the API.
+See [Notifications 2.0](https://cumulocity.com/guides/10.17.0/reference/notifications/) for more details about the API.
```
Comment (rssag) [inline content/microservice-sdk/java-bundle/client-library.md:263]: the reference guide is not included in the new design.
this should probably be an API link for now

File: content/microservice-sdk/java-bundle/client-library.md [hunk 2]
```diff
@@ -299,7 +300,7 @@ final NotificationTokenRequestRepresentation tokenRequestRepresentation = new No
         1440,                       // The token expiration duration in minutes.
         false);
 
-// The obtained token is required for establishing a WebSocket connection. Refer to [Notifications 2.0](/reference/notifications) in the *Reference guide* for more details.
+// The obtained token is required for establishing a WebSocket connection. Refer to [Notifications 2.0](https://cumulocity.com/guides/10.17.0/reference/notifications/) in the *Reference guide* for more details.
```
Comment (rssag) [inline content/microservice-sdk/java-bundle/client-library.md:303]:
```suggestion
// The obtained token is required for establishing a WebSocket connection. Refer to [Notifications 2.0](https://cumulocity.com/guides/10.17.0/reference/notifications/) for more details.
```
api link

File: content/microservice-sdk/rest-bundle/microservice-development.md [hunk 3]
```diff
@@ -441,9 +441,9 @@ Content-Type: application/vnd.com.nsn.cumulocity.managedobjectcollection+json; c
 }
 ```
 
-Now you could, for example, use the `c8y_Position` property to locate/pin the object on a map. Standard fragments are defined in the [Device integrator library](/reference/device-integrator-library/) and in the [Sensor library](/reference/sensor-library/).
+Now you could, for example, use the `c8y_Position` property to locate/pin the object on a map. Standard fragments are defined in the [fragment library](/device-integration/fragment-library/) and in the [sensor library](/concepts/domain-model/#sensor-library).
```
Comment (rssag) [inline content/microservice-sdk/rest-bundle/microservice-development.md:444]:
```suggestion
Now you could, for example, use the `c8y_Position` property to locate/pin the object on a map. Standard fragments are defined in the [fragment library](/device-integration/fragment-library/).
```
the sensor library no longer exists, it was partially integrated into the new fragment library and partially removed

i think the sensor library link can be removed entirely here

File: content/microservice-sdk/microservice-sdk-introduction.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Getting started
-slug: introduction
+slug: microservice-getting-started
```
Comment (BeateRixen) [inline content/microservice-sdk/microservice-sdk-introduction.md:3]: I think we can remove the slug. I would probably rename the title to "Introduction" instead.

File: content/microservice-sdk/rest-bundle/microservice-development.md [hunk 5]
```diff
@@ -486,7 +486,7 @@ Content-Type: application/vnd.com.nsn.cumulocity.measurementcollection+json; cha
 
 #### Sending operations to devices
 
-To trigger an operation on a device, POST the operation to the [Device Control API](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Device-control-API). The following example restarts the device with the ID "2480300" (which is the Raspberry Pi that is integrated in [Device integration](/device-integration/rest#device-integration) in *Device integration*.
+To trigger an operation on a device, POST the operation to the [Device Control API](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Device-control-API). The following example restarts the device with the ID "2480300" (which is the Raspberry Pi that is integrated in [Device integration](/device-integration/rest#device-integration).
```
Comment (BeateRixen) [inline content/microservice-sdk/rest-bundle/microservice-development.md:489]: The link looks weird. @rssag Do we still have this structure "device-integration/rest/#device-integration"? Probably you should better rename the device-integration file in the rest  bundle?

File: content/microservice-sdk/rest-bundle/rest-overview.md [hunk 6]
```diff
@@ -15,11 +15,11 @@ With this API description you will learn how to use {{< product-c8y-iot >}}'s RE
 The description is closely linked to the {{< openapi >}}, which describes each interface in detail. Relevant secions in the {{< openapi >}} are in particular:
 
 - [REST implementation](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#section/REST-implementation) is the reference for all general concepts.
-- [Device integrator library](/reference/device-integrator-library/) specifies the data model for device management.
-- [Sensor library](/reference/sensor-library/) specifies the data model for sensors and controls.
+- [Fragment library](/device-integration/fragment-library/) specifies the data model for device management.
+- [Sensor library](/concepts/domain-model/#sensor-library) specifies the data model for sensors and controls.
```
Comment (BeateRixen) [inline content/microservice-sdk/rest-bundle/rest-overview.md:19]: @rssag Can you please check with the DM team, what we should do with this chapter in the Concepts section? We have a couple of links pointing to this. I saw in the history that the team worked on this file when they created this fragment library document, not sure why they didn´t rename it.

**Comment by BeateRixen** [comment]
@MWindel Shouldn´t we take the opportunity to rename the concepts folder to "general-aspects"? We would have to search for this then throughout the docs, but since you rename the "manifest" file which is referenced quite often, we anyway have to so do. @rssag What do you think?

File: content/microservice-sdk/concept-bundle/microservice-utility-tool.md [hunk 7]
```diff
@@ -123,7 +123,7 @@ The successful execution will create an application on the {{< product-c8y-iot >
 
 If the name of the microservice application is not provided in the manifest file, it will be automatically inferred from the ZIP file name without the version number.
 
-For further information on deploying microservices to {{< product-c8y-iot >}}, refer to [Administration > Managing and monitoring microservices](/users-guide/administration#managing-microservices) in the *User guide*.
+For further information on deploying microservices to {{< product-c8y-iot >}}, refer to [Managing microservices](/standard-tenant/ecosystem/#managing-microservices) and [Monitoring microservices](/standard-tenant/ecosystem/#monitoring-microservices).
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/microservice-sdk/concept-bundle/microservice-utility-tool.md:126]:
```suggestion
For further information on deploying microservices to {{< product-c8y-iot >}}, refer to [Managing microservices](/standard-tenant/ecosystem/#managing-microservices).
```

File: content/microservice-sdk/cs-bundle/cs-overview.md [hunk 8]
```diff
@@ -1,78 +1,78 @@
...
+layout: redirect
+---
+
+This section describes how to develop and deploy microservices on top of {{< product-c8y-iot >}} using the Microservice SDK for C#, and it contains:
+
+*   [Prerequisites](#general-prerequisites) – Development and runtime requirements you must develop and run C# microservices.
+*   [Hello world tutorial for C#](#cs-hello-world-tutorial) – Step-by-step instructions to develop and deploy your first C# microservice.
+*   [Developing microservices](#developing-microservices) - Information about this SDK and detailed C# API reference.
+*   [Troubleshooting](#cs-troubleshooting) – Some identified common problems and their solutions.
+<!-- *   [Authentication](#sso-oauth) – Information about authentication employing OAuth tokens. -->
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/microservice-sdk/cs-bundle/cs-overview.md:13]:
```suggestion
<!-- *   [Authentication](#microservice-cs-authentication) – Information about authentication employing OAuth tokens. -->
```

File: content/microservice-sdk/microservice-sdk-introduction.md (hunk 4 above)
Comment (MWindel) [reply content/microservice-sdk/microservice-sdk-introduction.md:3]:
```suggestion

```

File: content/microservice-sdk/microservice-sdk-introduction.md [hunk 9]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Getting started
```
Comment (MWindel) [inline content/microservice-sdk/microservice-sdk-introduction.md:2]:
```suggestion
title: Introduction
```

**Comment by rssag** [comment]
> @MWindel Shouldn´t we take the opportunity to rename the concepts folder to "general-aspects"? We would have to search for this then throughout the docs, but since you rename the "manifest" file which is referenced quite often, we anyway have to so do. @rssag What do you think?

i'm indifferent. "general-aspects" is longer and hyphenated. "concepts" or "general" both work for me. "concepts" is the status quo and would avoid renaming.

File: content/microservice-sdk/cs-bundle/microservice-cs-authentication.md [hunk 10]
```diff
@@ -1,10 +1,11 @@
 ---
 weight: 40
-layout: redirect
 title: Authentication
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/microservice-sdk/cs-bundle/microservice-cs-authentication.md]: Done

File: content/microservice-sdk/java-bundle/client-library.md (hunk 1 above)
Comment (MWindel) [reply content/microservice-sdk/java-bundle/client-library.md:263]:
```suggestion
See [Notifications 2.0](https://cumulocity.com/api/core/#tag/Notification-2.0-API) for more details about the API.
```

File: content/microservice-sdk/java-bundle/client-library.md [hunk 11]
```diff
@@ -299,7 +300,7 @@ final NotificationTokenRequestRepresentation tokenRequestRepresentation = new No
         1440,                       // The token expiration duration in minutes.
         false);
 
-// The obtained token is required for establishing a WebSocket connection. Refer to [Notifications 2.0](/reference/notifications) in the *Reference guide* for more details.
+// The obtained token is required for establishing a WebSocket connection. Refer to [Notifications 2.0](https://cumulocity.com/guides/10.17.0/reference/notifications/) for more details.
```
Comment (MWindel) [inline content/microservice-sdk/java-bundle/client-library.md:303]:
```suggestion
// The obtained token is required for establishing a WebSocket connection. Refer to [Notifications 2.0](https://cumulocity.com/api/core/#tag/Notification-2.0-API) for more details.
```

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**